spec/handlers/newflow/change_password_spec.rb

```diff
@@ -27,19 +27,33 @@ module Newflow
     end
 
     context 'failure' do
-    let(:params) do
-      {
-        change_password_form: {
-          password: 'pwd',
-        }
-      }
-    end
+      before do
+        FactoryBot.create(:identity, user: user, password: 'password')
+      end
 
       it 'fails if password is too short' do
-        # create the password to begin with
-        FactoryBot.create(:identity, user: user, password: 'password')
+        params = {
```

Comment:
minor...this params construct (all thru the test) could be dryed up a bit thru a let(:params) and a let(:password) fields.

Reply:
Nice, I'd like to do that but I can't use `let(:password)` from within an example 🤔 .

Reply:
Done. I used nested `describe` blocks and `example do` blocks.